lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -114,7 +114,7 @@ sub _hmt_csv {
     my $hmt_ref = {};
 
     my $csv = Text::CSV->new({binary => 1}) or die "Cannot use CSV: " . Text::CSV->error_diag();
-    open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                    ## no critic (RequireBriefOpen)
+    open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                       ## no critic (RequireBriefOpen)
```

Comment:
Since we already include `Path::Tiny`, could we make use of its [`tempfile`](https://metacpan.org/pod/Path::Tiny#tempfile,-tempdir) here instead of the anonymous `open`?

Reply:
updated code use Path::Tiny to create temp file